astrokat/observe_main.py

```diff
@@ -312,7 +319,11 @@ def subarray_setup(self, instrument):
             conf_param = instrument[key]
             if key == "integration_time":
```

Comment:
Can't we remove this whole `if key == "integration_time"` block?  The yaml file loader always replaces the "integration_time" key with "dump_rate", so how would we end up with "integration_time" here?

Reply:
done